packages/shared/src/components/FeedLayout.tsx

```diff
@@ -163,6 +163,7 @@ const determineFeedSettings = ({
 
 export default function FeedLayout({
   children,
+  squadId,
```

Comment:
Hmm, is there not alternative way of passing data, not sure FeedLayout needs to have any knowledge of squads.
Technically it should be available under the page already right?

Reply:
We have different kinds of feed which is why I thought it is fitting to include it here since this is feed-related information.

It would be available to the post yes, though we will have to drill the props on multiple components, which using a Context solves.

Reply:
@idoshamun WDYT?

Reply:
Agree with Chris. I'm not sure why the feed layout has to change.
The mention feature is available only in the article modal and the article modal has access to source id already. Feed has nothing to do with this information.
Mentions need to be restricted in a general regardless if I'm in a squad feed or my feed but I opened a squad post

Reply:
> Mentions need to be restricted in a general regardless if I'm in a squad feed or my feed but I opened a squad post

Good point. So we won't have to do double work when we support squad posts in my-feed.